### PR TITLE
Feature/router ui

### PR DESCRIPTION
**Pull request by HTJin**
### Router UI
```
feat(RouterUi): Adds a navigation bar for web and mobile view

Installed material-ui, node-sass-chokidar, added icons, and lol@pitt logo
```

### CODE REVIEW

**Comment by Lewellync** [comment]
Going forward, do we want to have standardized commit messages or do you trust us to actually make sense of things? (obviously the first few don't need to make sense)

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
   render() {
     return (
-      <div className="App">
+      <BrowserRouter>
+        <div>
+          {/* WEB */}
+          <MediaQuery minDeviceWidth={1224}>
+            <Nav />
+          </MediaQuery>
+          <Route path='/' component={Home} />
```
Comment (zachcd) [inline src/App.js]: this should be an exact path

**Review by Lewellync — COMMENTED** [review]
Looks fine to me, a bunch of starter boilerplate code (although maybe the Nav.js/MobileNav.js should've been in their own pull?)

File: src/Pages/Pages.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#main {
+  display: flex;
+  justify-content: space-around;
+  img { margin-left: 0.1em; }
```
Comment (zachcd) [inline src/Pages/Pages.scss]: this should be identified by a class or id, not a tag

File: package.json [hunk 3]
```diff
@@ -3,10 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^1.3.1",
+    "@material-ui/icons": "^1.1.0",
+    "mdi-material-ui": "^5.2.0",
```
Comment (zachcd) [inline package.json:8]: whats this?

**Review by zachcd — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (HTJin) [reply package.json:8]: @material-ui/core are for the components and the other 2 are icon packages

**Comment by zachcd** [comment]
@Lewellync individual commits on a branch are low priority on clarity.  The branch itself should have a clear commit message in the PR and the branch should be squashed into one commit using Github's squash and merge.  That way the primary branch has a clear commit history.

File: package.json (hunk 3 above)
Comment (zachcd) [reply package.json:8]: cool sounds good.

**Review by zachcd — APPROVED** [review]

**Pull request merged**